template/src/__tests__/App-test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import App from '../App';
+import renderer from 'react/lib/ReactTestRenderer';
```

Comment:
Yes! Absolutely, I'm waiting for https://github.com/facebook/react/pull/7362 to land and will update all the Jest documentation asap.